### DISCUSSION TITLE
The VEAT forms (25, 26, 27, 28) contain almost no lot information in the result section (section 6)

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#671] [Q&A] [answered]
The VEAT forms (25, 26, 27, 28) contain almost no lot information in the result section (section 6). The reason that the information in the EFX templates is made dependent on OPT-322-LotResult. But OPT-322-LotResult is not included in the notices for notice 25, 26, 27 and 28. So the information can never be rendered. Can you correct this in an upcoming version of the SDK? 

For example 26.efx and in notice 26.json there is no OPT-322-LotResult  field. 
![image](https://github.com/OP-TED/eForms-SDK/assets/111063502/e391037a-7724-4f0a-92cf-75fcacff1d78)

This applies to all veat forms (25,26,27,28).

**Comment by rouschr** [comment]
Hello, 
please always state which SDK you are using for context reasons ("current SDK" can change from one day to another ...).
In SDK 1.8.0 and notice 26.json I find "OPT-322-LotResult"
https://github.com/OP-TED/eForms-SDK/blob/1.8.0/notice-types/26.json
I also find the "OPT-322-LotResult" in the view template, but maybe not where you expect it.

**Comment by kletersteegp** [comment]
Hello, thanks for the anwser. We use SDK1.7.1. Sory, I made a mistake in the field. It is about the OPT-320-LotResult field (not OPT-322-LotResult).  OPT-320-LotResult is used in the efx for for example form 26. But also in the link you added (
https://github.com/OP-TED/eForms-SDK/blob/1.8.0/notice-types/26.json) OPT-320-LotResult is not present.

**Reply by rouschr** [reply]
I will ask someone that is more into the rules logic, this is too close to the business for me.

**Reply by kletersteegp** [reply]
Ok Thanks. For extra information. The publication looks like this and there is missing detail information for LOT-1 and LOT-2. There are only a 6.1 headers without detail information.

![image](https://github.com/OP-TED/eForms-SDK/assets/111063502/d72f252f-f392-4589-a646-96757c957ef3)

**Reply by pdonohoe** [reply, marked as answer]
Dear kletersteegp,
Thank you for pointing out this bug. It will be fixed in a future release of the SDK, and the fix will be applied to previous versions of the SDK, so a new patch release of SDK 1.7 will be created.
